mapper/tedge_mapper/src/az_mapper.rs

```diff
@@ -0,0 +1,198 @@
+use crate::error::*;
+use crate::mapper::*;
+use crate::time_provider::TimeProvider;
+use mqtt_client::Topic;
+use thin_edge_json::serialize::ThinEdgeJsonSerializer;
+
+#[derive(Debug)]
+pub struct AzureMapperConfig {
+    pub in_topic: Topic,
+    pub out_topic: Topic,
+    pub errors_topic: Topic,
+}
+
+impl Default for AzureMapperConfig {
+    fn default() -> Self {
+        Self {
+            in_topic: Topic::new("tedge/measurements").unwrap(),
+            out_topic: Topic::new("az/messages/events/").unwrap(),
+            errors_topic: Topic::new("tedge/errors").unwrap(),
+        }
+    }
+}
+
+impl Into<MapperConfig> for AzureMapperConfig {
+    fn into(self) -> MapperConfig {
+        MapperConfig {
+            in_topic: self.in_topic,
+            out_topic: self.out_topic,
+            errors_topic: self.errors_topic,
+        }
+    }
+}
+
```

Comment:
```suggestion
impl From<AzureMapperConfig> for MapperConfig {
    fn from(val: AzureMapperConfig) -> Self {
        MapperConfig {
            in_topic: val.in_topic,
            out_topic: val.out_topic,
            errors_topic: val.errors_topic,
        }
    }
}
```

Reply:
Hmm I don't get what you meant. It's the same, isn't it?

Reply:
Ah, ok, we prefer `from` to `into`.